README.md

```diff
@@ -11,5 +11,5 @@ To generate /docs directory with parcel
 
 to avoid weird web socket with parcel:
 
-`parcel index.html  --out-dir docs/ --no-hmr`
+`parcel index.html  --out-dir docs/ --no-hmr NODE_ENV=production`
```

Comment:
Did you notice the `npm run build` script that I added, which uses `parcel build`? That seems to be the recommended method of building for production.

Reply:
@eemeli in the docs it is recommended to add `NODE_ENV=production` for production alaways

Reply:
Err, [this](https://parceljs.org/cli.html#build) is what I found in the Parcel docs:
> **Build**
> Builds the assets once, it also enabled minification and sets the `NODE_ENV=production` environment variable.

Reply:
@eemeli really? well, looks like I have either missed smth or just did not find it. Thanks!

Reply:
So, @eemeli shall we remove it and merge then?

Reply:
Yeah, that sounds good.